Geometry_Engine/Modify/CleanPolyline.cs

```diff
@@ -36,14 +36,19 @@ public static partial class Modify
         /**** Public Methods                            ****/
         /***************************************************/
 
-        [Description("Returns a polyline defined by the points which result in deviations from a straight line only. For example, if a polyline has 3 points in a straight line, the middle point is removed as part of this cleaning process. This is designed for closed polylines only")]
+        [Description("Returns a polyline defined by the points which result in deviations from a straight line only. For example, if a polyline has 3 points in a straight line, the middle point is removed as part of this cleaning process if distance betwen point is less than distanceTolerance. This is designed for closed polylines only")]
```

Comment:
The change to this description is fundamentally incorrect. The middle point is always removed in a straight line, regardless of the `distanceTolerance`

Reply:
Description still needs updating.

Reply:
Description still needs updating on this particular line